dd-opentracing-cpp/build/0.3.6/Dockerfile

```diff
@@ -5,7 +5,7 @@ RUN apt-get update && \
   # Basic circleci utils
   ca-certificates git curl wget tar sudo \
   # Source tools
-  clang-format-6.0 \
+  clang-format-6.0 clang-format-9 \
```

Comment:
Might as well remove `clang-format-6.0`, and for consistency, install `clang-9` instead of `clang-6.0` below.